packages/cli/src/userInteraction.ts

```diff
@@ -431,14 +435,16 @@ class CLIUserInteraction implements UserInteraction {
     if (config.validation || config.additionalValidationOnAccept) {
       validationFunc = async (input: string) => {
         let res: string | undefined = undefined;
+        const spinner = new CustomizedSpinner();
+        spinner.start();
```

Comment:
Make sure the UX is OK in terminals with different backgroud color: especially for white and black.

Reply:
The UX works for white, black, and blue background color